client/my-sites/post-share/index.jsx

```diff
@@ -0,0 +1,436 @@
+						comment: '%s will be replaced by a formatted price, i.e $9.99'
+					} )
+				}
+				description={ translate( 'Live chat support and no advertising.' ) }
+				list={ [
+					translate( 'Live chat support' ),
+					translate( 'No Advertising' )
+				] }
+				plan={ PLAN_BUSINESS }
+				title={ translate( 'Upgrade to a Business Plan!' ) }
+			/>
+		);
+	}
+
+	renderFooter() {
+		if ( ! this.isSchedulingEnabled ) {
+			return null;
+		}
+
+		const { postId, siteId, } = this.props;
+		const { selectedShareTab } = this.state;
+
+		return (
+			<div className="post-share__footer">
+				<SectionNav className="post-share__footer-nav" selectedText={ 'some text' }>
+					<NavTabs label="Status" selectedText="Published">
+						<NavItem
+							selected={ selectedShareTab === SCHEDULED }
+							count={ 4 }
+							onClick={ this.setFooterSection( SCHEDULED ) }
+						>
+							Scheduled
+						</NavItem>
+						<NavItem
+							selected={ selectedShareTab === PUBLISHED }
+							count={ 2 }
+							onClick={ this.setFooterSection( PUBLISHED ) }
+						>
+							Published
+						</NavItem>
+					</NavTabs>
+				</SectionNav>
+
+				<ActionsList
+					section={ selectedShareTab }
+					postId={ postId }
+					siteId={ siteId }
+				/>
+			</div>
+		);
+	}
+
+	render() {
+		if ( ! this.props.isPublicizeEnabled ) {
+			return null;
+		}
+
+		if ( ! this.props.planHasRepublicizeFeature ) {
+			return ( <div className="post-share">
+				<Banner
+					feature="republicize"
+					title={ this.props.translate( 'Unlock the ability to re-share posts to social media' ) }
+					callToAction={ this.props.translate( 'Upgrade to Premium' ) }
+					description={ this.props.translate( 'Get unlimited premium themes, video uploads, monetize your site and more.' ) }
+				/>
+			</div> );
+		}
+
+		const classes = classNames( 'post-share__wrapper', {
+			'has-connections': this.hasConnections()
+		} );
+
+		const {
+			connections,
+			failure,
+			postId,
+			requesting,
+			siteId,
+			siteSlug,
+			success,
+			translate,
+		} = this.props;
+
+		if ( ! siteId || ! postId ) {
+			return null;
+		}
+
+		return (
+			<div className="post-share">
+				{ requesting &&
+					<Notice status="is-warning" showDismiss={ false }>
+							{ translate( 'Sharing…' ) }
+					</Notice>
+				}
+
+				{ success &&
+					<Notice status="is-success" onDismissClick={ this.dismiss }>
+						{ translate( 'Post shared. Please check your social media accounts.' ) }
+					</Notice>
+				}
+
+				{ failure &&
+					<Notice status="is-error" onDismissClick={ this.dismiss }>
+						{ translate( 'Something went wrong. Please don\'t be mad.' ) }
+						</Notice>
+				}
+
+				<div className={ classes }>
+					<QueryPublicizeConnections siteId={ siteId } />
+					<QueryPostTypes siteId={ siteId } />
+
+					<div className="post-share__head">
+						<h4 className="post-share__title">
+							{ translate( 'Share this post' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Share this post!' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).